packages/jest-resolve/src/__tests__/resolve.test.ts

```diff
@@ -707,3 +707,51 @@ describe('Resolver.getModulePaths() -> nodeModulesPaths()', () => {
     expect(dirs_actual).toEqual(expect.arrayContaining(dirs_expected));
   });
 });
+
+describe('findGlobalPaths', () => {
```

Comment:
can be removed I think, it's kind of internal and we're testing it through Resolver.getGlobalPaths